ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js

```diff
@@ -673,124 +540,28 @@ export default class ConfirmApproveContent extends Component {
         <div className="confirm-approve-content__description">
           {this.renderDescription()}
         </div>
```

Comment:
There are still references to `ERC20` in other confirmation pages, not sure if that also needs caleanup.